packages/runtime/src/lib/pycompile-op.ts

```diff
@@ -90,6 +92,18 @@ export class PyCompileOp {
 		return spawnSync("python3", [filePath, scInitParam], { encoding: "utf8" });
 	}
 
+	/**
+	 * Description: This method checks if given module is installed or not.
```

Comment:
add information about the exception here `if given module is installed, otherwise throws exception...`

Reply:
```suggestion
	 * Method checks if given module is installed or not. exception info...
```